<launch>
    <rosparam file="$(find hoverpibot_hardware_interface)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find hoverpibot_hardware_interface)/config/controllers.yaml" command="load"/>
    <rosparam file="$(find hoverpibot_hardware_interface)/config/joint_limits.yaml" command="load"/>
    <node name="hoverpibot_hardware_interface" pkg="hoverpibot_hardware_interface" type="hoverpibot_hardware_interface_node" output="screen"/>
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/"
        args="
            /hoverpibot/controller/state
            /hoverpibot/controller/position/left_wheel
            /hoverpibot/controller/position/right_wheel
        "/>
</launch>
